frontend/package.json

```diff
@@ -1,7 +1,8 @@
 {
-  "name": "hello-world-app",
+  "name": "kmap-frontend",
```

Comment:
> The backed is also currently beeing renames. Adapt to this naming whatever it will look like in the end. See [#69 (comment)](https://github.com/amosproj/amos-ss2021-project2-context-map/pull/69#discussion_r630144092)
> 
> So this needs to be either `kmap frontend` or `kmap.frontend` for whichever version we decise.

package name's shouldn't contain spaces ([see here](https://docs.npmjs.com/cli/v7/configuring-npm/package-json)), so I'll rename it to `kmap.frontend`